Source/Fuse.Navigation/Navigation.Locators.uno

```diff
@@ -26,11 +26,10 @@ namespace Fuse.Navigation
 			if (t != null)
 				return t;
 				
-			for (int i = 0; i < node.Children.Count; i++)
+			for (var x = node.FirstChild<Behavior>(); x != null; x = x.NextSibling<Behavior>())
 			{
-				var c = node.Children[i] as IBaseNavigation;
-				//mean to check only behaviours (as was a distinct list before), so exclude visuals
-				if (c != null && !(c is Visual)) return c;
+				var c = x as IBaseNavigation;
```

Comment:
The template argument doesn't need to be T to call `FirstChild<T>` though. It can be any type. It's only if you need iteration it needs to be a `Node`, but that doesn't affect `FirstChild`.  Actually, it doesn't even affect `NextSibling<T>` either. It's only when used together in a for-loop they need to be nodes, but that should just work since the for-loop knows the type.

I'm pointing this out since I know there's at least one case on master nor where `FindFirst` is expected to work with non-node types.